### ISSUE TITLE
Using with NX,  error running nodejs ? 

### ISSUE THREAD

**Issue by appsolutegeek** [nestjs/ng-universal#82]
<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[x] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
I am a little confused here and there doesn't seem to be much docs. I have 2 apps configured in NX, one an angular and one the nestjs with SSR

I build both apps (using ng build) and then i get 

```
Error: Cannot find module '/Users/ian/Development/Personal/typescript/ui/dist/apps/api/main.js'
    at webpackEmptyContext (/Users/ian/Development/Personal/typescript/ui/dist/apps/api/webpack:/apps/api/src/app sync:2:1)
    at /Users/ian/Development/Personal/typescript/ui/dist/apps/api/webpack:/apps/api/src/app/app.module.ts:12:15
```

And my main app.module inside nestjs app i haver 

```
  imports: [
    AngularUniversalModule.forRoot({
      viewsPath: join(process.cwd(), "dist/apps/ui"),
      bundle: require(join(process.cwd(), "dist/apps/api/main.js"))
    })
  ],
```

I know that the files exist inside the dist folder.

Its not my first time with nestjs or angular but this is really confusing me. There doesn't seme to be any docs as such. I tested the example, and it has a webpack file in the root, i didn;t use that as NX builds my projects (node) i.e from angular.json

```
    "api": {
      "root": "apps/api",
      "sourceRoot": "apps/api/src",
      "projectType": "application",
      "prefix": "api",
      "schematics": {},
      "architect": {
        "build": {
          "builder": "@nrwl/builders:node-build",
          "options": {
```

So basically I am doing an

```
ng build api
ng build ui
```

and then running node dist/apps/api/main.js

but it fails..

## Expected behavior
To provide a clear example with an explanation of what each part is doing, would be really nice to comment about the use with NX.

## Environment

<pre><code>
Nest version: 6.1
 
For Tooling issues:
- Node version: 11
- Platform:  MAC
Others:

</code></pre>

**Comment by appsolutegeek** [comment]
Also doing an ng serve api, produces the following error, if I remove the AngularIUniversalModule from the app.module then ng serve works.

```
/Users/ian/Development/Personal/typescript/ui/dist/apps/api/webpack:/apps/api/src/app sync:2
        var e = new Error("Cannot find module '" + req + "'");
^
Error: Cannot find module '/Users/ian/Development/Personal/typescript/ui/dist/apps/api/main.js'
    at webpackEmptyContext (/Users/ian/Development/Personal/typescript/ui/dist/apps/api/webpack:/apps/api/src/app sync:2:1)
    at /Users/ian/Development/Personal/typescript/ui/dist/apps/api/webpack:/apps/api/src/app/app.module.ts:12:15
    at Object../apps/api/src/app/app.module.ts (/Users/ian/Development/Personal/typescript/ui/dist/apps/api/webpack:/apps/api/src/app/app.module.ts:18:26)
```

**Comment by twittwer** [comment]
I had the same problem. I've used a workaround with try-catch, because I do not need SSR in dev (serve) mode.
```ts
function getUniversalModule(): Array<any> {
  if (!environment.production) {
    return [];
  }
  try {
    return [
      AngularUniversalModule.forRoot({
        viewsPath: join(process.cwd(), 'dist', 'apps', 'frontend'),
        bundle: require('../../../../dist/apps/frontend-server/main.js'),
      }),
    ];
  } catch (error) {
    if (!environment.production) {
      return [];
    }
    throw error;
  }
}

@Module({
  imports: [
    ...getUniversalModule(),
  ],
})
export class AppModule {}
```

**Comment by appsolutegeek** [comment]
thanks @twittwer , can I ask you, did you use teh webpackconfig from the example code ?

Or did you just do a normal NX creation i.e UI and API - nothing special - right ?

**Comment by twittwer** [comment]
Correct, currently I'm not using any special webpack config

**Comment by servrox** [comment]
@appsolutegeek I created a working [demo](https://github.com/servrox/nx-ng-nest-universal) plus an explanation how to set it up. Maybe this helps you :)

**Comment by appsolutegeek** [comment]
@servrox Nice! Thank you, i am going to take a look at it tomorrow. This should save me some time :-)

**Comment by kamilmysliwiec** [comment]
Please, report this issue in the NX repo.
Also, thanks @servrox for great sample project!!
